mode/dockerfile/dockerfile.js

```diff
@@ -11,35 +11,64 @@
 })(function(CodeMirror) {
   "use strict";
 
+  var from = "from";
+  var fromRegex = new RegExp("^(\\s*)\\b(" + from + ")\\b", "i");
```

Comment:
Does changing this from a simple inline regexp to this complicated definition add anything?

Reply:
It seemed more consitent with the other instructions but I can change it back. Let me know.

Reply:
It's harmless, I guess, but it seemed a bit of a pointless change.